src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] COLORS = {"red", "blue", "green", "yellow", "purple", "orange"};
+
+    public String getRandomColor() {
+        Random random = new Random();
```

Comment:
Consider making 'random' a private final field of the class instead of instantiating it in the 'getRandomColor' method. This way, you avoid creating a new instance of Random every time the method is called, which is more efficient.